packages/terra-clinical-site/src/examples/action-header/Index.jsx

```diff
@@ -24,6 +25,8 @@ const HeaderExamples = () => (
     <ActionHeaderBackNextPreviousClose />
     <h2>Maximize, Close</h2>
     <ActionHeaderExpandClose />
+    <h2>Close, keepCloseButton</h2>
```

Comment:
Non-technical audience may not understand what this example does. Perhaps rephrase keepCloseButton to something that is a bit more descriptive.

Reply:
you mean I should change the [property name](https://github.com/cerner/terra-clinical/pull/273/files#diff-bb6aa0f3cc7f39549fc68570eab3f3c4R63) or just add explanation in this example?

Reply:
Just the description in the example. Instead of "Close, keepCloseButton", perhaps "Keeps Close button in small view port" or something similar.